pkg/builder/builder.go

```diff
@@ -1308,7 +1308,7 @@ func BuildSandboxConfigMap(nm types.NamespacedName, vdb *vapi.VerticaDB, sandbox
 		// the data should be immutable since dbName and sandboxName are fixed
 		Immutable: &immutable,
 		Data: map[string]string{
-			vapi.VerticaDBNameKey: vdb.Spec.DBName,
+			vapi.VerticaDBNameKey: vdb.Name,
```

Comment:
If the configmap name is <vdbname>-<sandbox> we don't need to pass the namespaced name

Reply:
The VerticaDB and ConfigMap are assumed to be in the same namespace.

Reply:
I meant e configmap name is <vdbname>-<sandbox> we don't need to pass the namespaced name at 1294.